components/encoder/incremental/incremental_encoder.go

```diff
@@ -226,9 +226,6 @@ func (e *Encoder) Start(ctx context.Context, b board.Board) {
 	e.activeBackgroundWorkers.Add(1)
 
 	utils.ManagedGo(func() {
-		// Remove the callbacks added by the interrupt stream.
-		defer e.A.RemoveCallback(ch)
-		defer e.B.RemoveCallback(ch)
```

Comment:
Not to myself: we don't need these any more because the callbacks will get removed when `e.cancelCtx` is canceled.